galley/pkg/config/schema/schema.go

```diff
@@ -23,6 +23,18 @@ import (
 	"istio.io/istio/galley/pkg/config/schema/collection"
 )
 
+// Well-known snapshot names
```

Comment:
Should these be generated when we run `go generate` on metadata.yaml? I worry that hand-defining these is just asking for trouble down the line when we inevitably get out of sync.